README.md

```diff
@@ -182,6 +202,60 @@ function App () {
 }
 ```
 
+### Subscription (e.g.: socket.io)
```

Comment:
Should this code be inside the examples folder?

Reply:
@sergiodxa Yeah, I think we can add this model to your localStorage example using the `storage` event. So we can sync the data without focusing on the other window.

Reply:
Setting up socket.io for an example might be a bit overkill IMO.